core/client/fs/src/main/java/alluxio/client/meta/MetaMasterClient.java

```diff
@@ -35,7 +35,7 @@
    * @param options backup options
    * @return the server response
    */
-  BackupResponse backup(BackupPOptions options) throws IOException;
+  BackupPResponse backup(BackupPOptions options) throws IOException;
```

Comment:
will this break client compatibility?

Reply:
Yes.

Reply:
@apc999 are we ok with this client change?